configs/gunicorn.py

```diff
@@ -0,0 +1,37 @@
+from os import getenv
+from multiprocessing import cpu_count
+
+bind = f"0.0.0.0:{getenv('APP_PORT', '8000')}"
+
+# Log section
+if getenv('DEBUG') == "true":
+    loglevel = 'debug'
+else:
+    loglevel = 'info'
+
+accesslog = '-'
+errorlog = '-'
+access_log_format = '%(h)s %(l)s %(u)s %(t)s \"%(r)s\" %(s)s %(b)s \"%(f)s\" \"%(a)s\" %({X_Forwarded_For}i)s"'
+
+# How long to keep connection alive, this should be timed
+# a little less than the ELB idle timeout
+keepalive = getenv('APP_GUNICORN_TIMEOUT', '118')
```

Comment:
Does this need to be an int or OK as a string?

Reply:
It didn't seem to care if it was an int or a string when i tested it out